app/assets/stylesheets/static.scss

```diff
@@ -0,0 +1,84 @@
+// Place all the styles related to the static controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+@import "variables";
+@import "agency";
```

Comment:
this is ok, but adding `@imports` to several files + rails `require tree` is error prone.
First it makes dependencies harder to follow and it could lead to circular references, check [bootstrap](https://github.com/twbs/bootstrap/blob/master/less/bootstrap.less) for a good example